src/util/getPageCategory.ts

```diff
@@ -0,0 +1,20 @@
+// TODO: Move this data to our i18n system to support localized category labels.
+const defaultCategory = 'Learn';
+const categories = [
+	['tutorial/', 'Tutorials'],
+	['reference/', 'Reference'],
+	['guides/deploy/', 'Deploy Guides'],
+	['guides/integrations-guide/', 'Integration Reference'],
```

Comment:
```suggestion
	['guides/integrations-guide/', 'Integration Guides'],
```

Amazing, @delucis! SO excited for this search categorization!  🙌 

I think I'd prefer "Integration Guides" here, to be more in keeping with the file path/URL and the tab they'll show up under? But LGTM!